AppProject/Client/Pages/Homepage.razor

```diff
@@ -0,0 +1,25 @@
+﻿@page "/"
+@inherits BasePage
+@using AppProject.Client.Components;
+@using AppProject.Client.Shared;
+
+<div style="display:flex; justify-content:space-evenly; margin-top:1rem;">
+    <MudButton Variant="Variant.Filled" Class="rounded-circle" OnClick="() => NavigateToList(Platform.PC)">
+        <img style="height:3rem; width:2rem;" src="../icons/computer-monitor.svg" />
+    </MudButton>
+    <MudButton Variant="Variant.Filled" Class="rounded-circle" OnClick="() => NavigateToList(Platform.Playstation)">
+        <img style="height:3rem; width:2rem;" src="../icons/playstation.svg" />
+    </MudButton>
+    <MudButton Variant="Variant.Filled" Class="rounded-circle" OnClick="() => NavigateToList(Platform.Xbox)">
+        <img style="height:3rem; width:2rem;" src="../icons/xbox.svg" />
+    </MudButton>
+    <MudButton Variant="Variant.Filled" Class="rounded-circle" OnClick="() => NavigateToList(Platform.Switch)">
+        <img style="height:3rem; width:2rem;" src="../icons/nintendo-switch.svg" />
+    </MudButton>
+    <MudButton Variant="Variant.Filled" Class="rounded-circle" OnClick="() => NavigateToList()">
+        <div style="height:3rem; width:2rem; padding-top:40%; font-weight:700;">All</div>
+    </MudButton>
+</div>
+
+<HorizontalProductScroller Class="mt-5" Title="Popular Games" Items="items"></HorizontalProductScroller>
```

Comment:
Filter auf die Items??
Machen wir das später?
Die Daten in die DB sind dafür noch zu wenige.

Reply:
Brauchen dafür erstmal Bewertungen oder sowas, daher würde ich das machen sobald diese Dinge da sind.